tests/10-issue.js

```diff
@@ -26,95 +26,325 @@ describe('BitstringStatusList Credentials (Issue "statusPurpose: revocation")',
+                    'Expected statusSize to be >= 0.');
+                });
+                if(issuedVc.credentialStatus.statusSize.gt(1)) {
+                  it('If statusSize is provided and is greater than 1, ' +
+                    'then the property credentialStatus.statusMessage ' +
+                    'MUST be present',
+                  async function() {
+                    this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20statusSize%20is%20provided%20and%20is%20greater%20than%201%2C%20then%20the%20property%20credentialStatus.statusMessage%20MUST%20be%20present';
+                    issuedVc.credentialStatus.should.have.own.property(
+                      'statusMessage');
+                  });
+                }
+                it('The number of status messages MUST equal the number ' +
+                  'of possible values.',
+                async function() {
+                  this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=the%20number%20of%20status%20messages%20MUST%20equal%20the%20number%20of%20possible%20values';
+                  issuedVc.credentialStatus.statusSize.should.be.an('array');
+                  issuedVc.credentialStatus.statusSize.length.should.be.equal(
+                    issuedVc.credentialStatus.statusSize);
+                });
+                it('If present, the statusMessage property MUST be an array, ' +
+                  'the length of which MUST equal the number of possible ' +
+                  'status messages indicated by statusSize',
+                async function() {
+                  this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20present%2C%20the%20statusMessage%20property%20MUST%20be%20an%20array%2C%20the%20length%20of%20which%20MUST%20equal%20the%20number%20of%20possible%20status%20messages%20indicated%20by%20statusSize';
+                  issuedVc.credentialStatus.statusSize.should.be.an('array');
+                  issuedVc.credentialStatus.statusSize.length.should.be.equal(
+                    issuedVc.credentialStatus.statusSize);
+                });
+                if(issuedVc.credentialStatus.statusSize.gt(1)) {
+                  it('statusMessage MAY be present if statusSize is 1, ' +
+                  'and MUST be present if statusSize is greater than 1.',
+                  async function() {
+                    this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=statusMessage%20MAY%20be%20present%20if%20statusSize%20is%201%2C%20and%20MUST%20be%20present%20if%20statusSize%20is%20greater%20than%201';
+                    issuedVc.credentialStatus.should.have.own.property(
+                      'statusMessage');
+                  });
+                }
+                it('If the statusMessage array is present, each ' +
+                  'element MUST contain the two properties described below',
```

Comment:
someone is definitely going to ask what those 2 properties are.

Reply:
A line (or two?) seem to have been lost, as this `it()` should end with a `.` which should be preceded by something like `and MAY contain additional properties`, based on the `this.test.link` in (current) line 196, `'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20the%20statusMessage%20array%20is%20present%2C%20each%20element%20MUST%20contain%20the%20two%20properties%20described%20below%2C%20and%20MAY%20contain%20additional%20properties.'`.

I agree that "the two properties described below" should be explicitly identified here (and probably in the spec), as "below" is fairly meaningless in *this* context.

Reply:
Added the 2 properties in the statement